ORStools/proc/isochrones_layer_proc.py

```diff
@@ -82,9 +86,22 @@ def __init__(self):
                 name=self.IN_RANGES,
                 description="Comma-separated ranges [min or m]",
                 defaultValue="5, 10"
-            )
+            ),
+            QgsProcessingParameterNumber(
+                name=self.IN_SMOOTHING,
+                description="Smoothing factor between 0 [fine grained] and 100 [concave hull]",
```

Comment:
Where does `[concave hull]` come from? Are you sure about this?

Reply:
Not sure as i didn't add this. Should we change it to `Smoothing factor between 0 [particularized] and 100 [generalized]`? The documentation in the API Playground doesn't state anything regarding concave hull.

Reply:
I'd propose `[detailed]` and `[generalized]`.